src/container/chrome/chrome-container.tsx

```diff
@@ -70,6 +71,14 @@ export const ChromeContainer = MobxReact.inject('store')(
 						title={project ? project.getName() : 'Alva'}
 					/>
 				)}
+				<BugReport
+					title="Found a bug?"
+					onClick={() => {
+						Electron.shell.openExternal(
```

Comment:
We are in the process of moving all `Electron` dependencies to the main process. We should move this call to `electron/main.js` and send an event to trigger it instead

Reply:
Sounds like a good idea, but how would I do that @marionebl?

Reply:
Got an idea. Trying it out now.